### DISCUSSION TITLE
Generate Graphs more often than once per day?

### DISCUSSION THREAD

**Discussion by snxox** [upptime/upptime#661] [Q&A] [answered]
Hi guys,

I wanna know whether it's possible to run the other tasks also more than once per day. is it possible to generate a graph once every 5 minutes?
I found the cron file in .github but the system is overwriting it cionstantly.

Btw is it possible therefore to create my own bot like the Upptime bot who runs that graph stuff once every 5 minutes?

Thank you very much.

**Comment by AnandChowdhary** [comment, marked as answer]
You can edit it like so: https://upptime.js.org/docs/configuration#ci-schedule
